framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/WorkbenchWithKieServerScenarioImpl.java

```diff
@@ -128,9 +133,19 @@ public WorkbenchWithKieServerScenarioImpl(OpenShiftController openShiftControlle
 
         logger.info("Waiting for Kie server to register itself to the Workbench.");
         KieServerControllerClientProvider.waitForServerTemplateCreation(workbenchDeployment);
+
+        databaseDeployment = new DatabaseDeploymentImpl();
+        databaseDeployment.setOpenShiftController(openshiftController);
+        databaseDeployment.setNamespace(projectName);
+        databaseDeployment.setDatabase("mysql");
```

Comment:
Database name could be provided by property, can be for example defined in test maven module.